sql/moz-fx-data-shared-prod/monitoring_derived/bigquery_tables_inventory_v1/query.py

```diff
@@ -21,18 +21,64 @@
 parser.add_argument("--source_projects", nargs="+", default=DEFAULT_PROJECTS)
 parser.add_argument("--destination_dataset", default="monitoring_derived")
 parser.add_argument("--destination_table", default="bigquery_tables_inventory_v1")
+parser.add_argument("--tmp_table", default="bigquery_tables_last_modified_tmp")
 
 
-def create_query(source_project):
+def create_last_modified_tmp_table(project, tmp_table_name):
+    """Create temp table to capture last modified dates."""
+    # remove old table in case of re-run
+    client = bigquery.Client(project)
+    client.delete_table(tmp_table_name, not_found_ok=True)
```

Comment:
Would it be easier to just always overwrite the table or would there be any value in keeping this table around with historical entries?

Reply:
Historical entries would be included in the destination table.